tests/use_cases/test_paytree_child_pair_monotonic_cas.py

```
@@ -0,0 +1,147 @@
+    status, _ = await _save(repo, _make_channel("c1", max_steps=7, last_ref=None), 1)
+    assert status == 1
+
+    assert "c1" in await store.zrevrange("payment_channels:all", 0, -1)
+    assert "c1" in await store.zrevrange("payment_channels:open", 0, -1)
+    assert "c1" not in await store.zrevrange("payment_channels:closed", 0, -1)
+
+
+@pytest.mark.asyncio
+async def test_child_pair_node_keys_are_plain_k_strings() -> None:
+    """Node keys are `str(2k)`/`str(2k+1)` — no depth/level math required to store them."""
+    store, repo = await _new_repo()
```

Ruff (0.16.1)

[warning] Unpacked variable `store` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)